### ISSUE TITLE
Compatible with tf.keras? 

### ISSUE THREAD

**Issue by innat** [CyberZHG/keras-gradient-accumulation#8]
I think it's not fully compatible with `tf.keras`. Can you confirm? I've faced an error with that. In that case I like to send PR for it.

**Comment by CyberZHG** [comment]
It works normally on my machine with `Keras==2.4.3` and `tensorflow==2.4.0`. Remember to add `TF_KERAS=1` to your environment variables if you want to use `Keras` in `TensorFlow` with this repo.

**Comment by CyberZHG** [comment]
The specification of the optimizers has been changed after 2.4.
